### PR TITLE
Manage resources

### PR DESCRIPTION
**Pull request by anushka-sinha**
manage resource screen ui

### CODE REVIEW

File: src/WebUI/ClientApp/src/app/feature/manage-resources/manage-resources.component.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+export class ManageResourcesComponent implements OnInit{
+    @ViewChild("fileInput", {static: false}) fileInput: ElementRef;
+    HRMSFile: any  = [];  
+    allocationFile: any = [];
+    synechronFile: any = [];
+    udemyFile: any = [];
+    displayedColumns = ['serialNo','fileRecordCount', 'error','status', 'path']
+    executionDatasource= new MatTableDataSource<any>()
+    fakepath : string = 'C:\\fakepath\\';
```
Comment (synechron-gm) [inline src/WebUI/ClientApp/src/app/feature/manage-resources/manage-resources.component.ts:19]: please change variable name and path 
Path would be : .\\ManageResourceFiles

File: src/WebUI/ClientApp/src/app/feature/manage-resources/manage-resources.component.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
+<div class="main-content table-padding">
```
Comment (synechron-gm) [inline src/WebUI/ClientApp/src/app/feature/manage-resources/manage-resources.component.html:1]: Can we have one single upload child component or directives?

File: src/WebUI/ClientApp/src/app/feature/manage-resources/manage-resources.component.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
+import { Component, OnInit, ViewChild, ElementRef } from '@angular/core'
```
Comment (synechron-gm) [inline src/WebUI/ClientApp/src/app/feature/manage-resources/manage-resources.component.ts:1]: Can we show toaster message on upload successful ?

Comment (anushka-sinha) [reply src/WebUI/ClientApp/src/app/feature/manage-resources/manage-resources.component.ts:1]: Yes Mandar,we can show that after successful api call.But currently,api has not been integrated.that why it is not being shown

File: src/WebUI/ClientApp/src/app/feature/manage-resources/manage-resources.component.ts (hunk 1 above)
Comment (anushka-sinha) [reply src/WebUI/ClientApp/src/app/feature/manage-resources/manage-resources.component.ts:19]: Okay

File: src/WebUI/ClientApp/src/app/feature/manage-resources/manage-resources.component.html (hunk 2 above)
Comment (anushka-sinha) [reply src/WebUI/ClientApp/src/app/feature/manage-resources/manage-resources.component.html:1]: Yes it can be done..but will take little time

**Pull request merged**